### PR TITLE
Clarify tiny timeouts behavior and how it can affect performance

### PR DESCRIPTION
**Pull request by asvetlov**
See also https://github.com/aio-libs/aiohttp/discussions/6008

### CODE REVIEW

File: CHANGES/6274.doc.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Describe how tiny timeouts can affect performance.
```
Comment (webknjaz) [inline CHANGES/6274.doc.rst]:
```suggestion
Added an explanation of how tiny timeouts affect performance to the client reference document.
```

File: docs/client_reference.rst [hunk 2]
```diff
@@ -1710,6 +1710,31 @@ ClientTimeout
...
+      E.g., assume a timeout is ``10``, absolute time when timeout should expire
+      is ``loop.time() + 5``, and it points to ``12345.67 + 10`` which is equal
+      to ``12355.67``.
+
+      The absolute time for the timeout cancellation is ``12356``.
+
+      It leads to grouping all close scheduled timeout expirations to exactly
+      the same time to reduce amount of loop wakeups.
+
+      .. versionchanged:: 3.7
```
Comment (webknjaz) [inline docs/client_reference.rst:1729]: Should this be dedented?

File: CHANGES/6274.doc.rst (hunk 1 above)
Comment (asvetlov) [reply CHANGES/6274.doc.rst]: Thanks. Forgot about the past tense.

File: docs/client_reference.rst (hunk 2 above)
Comment (asvetlov) [reply docs/client_reference.rst:1729]: Why not? The change was introduced by aiohttp 3.7

**Pull request merged**